helium/src/scss/_site_footer.scss

```diff
@@ -49,7 +49,7 @@
     min-height: 56px;
   }
 
-  a.as_admin, #course_settings_link {
```

Comment:
this should not be removed, as this button will still be required top be shown to admins when clicking on links directly to the course/lesson pages.